metric/collector/database_count.go

```diff
@@ -0,0 +1,53 @@
+package collector
+
+import (
+	"context"
+	"strconv"
+
+	metricAPI "github.com/bytebase/bytebase/metric"
+	"github.com/bytebase/bytebase/plugin/metric"
+	"github.com/bytebase/bytebase/store"
+	"go.uber.org/zap"
+)
+
+var _ metric.Collector = (*databaseCountCollector)(nil)
+
+// databaseCountCollector is the metric data collector for database.
+type databaseCountCollector struct {
+	l     *zap.Logger
+	store *store.Store
+}
+
+// NewDatabaseCountCollector creates a new instance of databaseCountCollector
+func NewDatabaseCountCollector(l *zap.Logger, store *store.Store) metric.Collector {
+	return &databaseCountCollector{
+		l:     l,
+		store: store,
+	}
+}
+
+// Collect will collect the metric for database
+func (c *databaseCountCollector) Collect(ctx context.Context) ([]*metric.Metric, error) {
+	var res []*metric.Metric
+
+	databaseCountMetricList, err := c.store.CountDatabaseGroupByBackupScheduleAndEnabled(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, databaseCountMetric := range databaseCountMetricList {
+		labels := map[string]string{
+			"backup_schedule": string(databaseCountMetric.BackupPlanPolicySchedule),
+		}
+		if databaseCountMetric.BackupSettingEnabled != nil {
+			labels["backup_enabled"] = strconv.FormatBool(*databaseCountMetric.BackupSettingEnabled)
```

Comment:
We'd better also set `"false"` for `backup_enabled` when `databaseCountMetric.BackupSettingEnabled` is `nil` to align the metric data

Reply:
"no backup setting" is different from "backup is not enabled" IMHO.
changed to `"null"` when no setting